src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -194,7 +193,6 @@ MavlinkMissionManager::update_geofence_count(unsigned count)
 		_count[MAV_MISSION_TYPE_FENCE] = count;
 
 	} else {
-		PX4_ERR("WPM: can't save mission state");
```

Comment:
send_statustext_critical() does not repeat the string, so the above was duplicate.